client/src/components/ServicesWorkspace.module.css

```
@@ -63,6 +63,148 @@
+
+.chapterNotesClose:hover {
+  background: rgb(255 255 255 / 8%);
+  color: var(--text-primary);
+}
+
+.chapterNotesSheetBody {
+  overflow-y: auto;
+  padding: 1.5rem 1.8rem 1.8rem;
+}
+
+.chapterNotesContent :global(.chapter-note-list),
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)